addOns/spider/src/main/java/org/zaproxy/addon/spider/SpiderController.java

```diff
@@ -97,6 +109,54 @@ protected SpiderController(Spider spider, List<SpiderParser> customParsers) {
         for (SpiderParser parser : customParsers) {
             this.addSpiderParser(parser);
         }
+
+        SpiderParam spiderParam = spider.getSpiderParam();
+        ParameterParser parser = DEFAULT_PARAMETER_PARSER;
+        bodyNormalizer =
+                rsrc -> {
+                    String body = rsrc.getBody();
+                    if (body.isEmpty()) {
+                        return body;
+                    }
+
+                    if (!rsrc.getHeaders().isEmpty()) {
+                        String contentType =
+                                rsrc.getHeaders().stream()
+                                        .filter(
+                                                e ->
+                                                        HttpHeader.CONTENT_TYPE.equalsIgnoreCase(
+                                                                e.getName()))
+                                        .findFirst()
+                                        .map(HttpHeaderField::getValue)
+                                        .map(e -> e.toLowerCase(Locale.ROOT))
+                                        .orElse(null);
+                        if (contentType != null
+                                && !contentType.contains(HttpHeader.FORM_URLENCODED_CONTENT_TYPE)) {
```

Comment:
Nit for this PR but should this method be handling multipart as well?

Reply:
The spider does not support multipart.